File: PRAK304_2210817110001_AkhmadRaihanRidha.py
A= int (input("Sebuah baris berisi satu buah bilangan cacah dan batas maksimal 99.(a>=0<100) :"))

print("Sebuah baris berisi sebuah bilangan cacah yang merupakan Ejaan dari bilangan tersebut atau batas limit dari bilangan")


if A < 10 and A>0:
    print ("Satuan")
elif A > 10 and A < 20 :
    print("Belasan")   
elif A == 0 :
    print("Nol")   
elif A > 20 and A < 99 :
    print("Puluhan")
elif A > 99 :
    print ("Anda Menginput Melebihi Limit Bilangan")

File: PRAK304_2210817110001_AkhmadRaihanRidha.c
#include <stdio.h>
int main(){
    int bil;
    printf("sebuah baris berisi satu buah bilangan cacah");
    printf(" dan batas maksimal 99(a>=0<100)\t : ");
    scanf("%i",&bil);
    if (bil>0 && bil<10)
    {
        printf("satuan");
    }
    else if (bil==0){
        printf("Nol");
    }
    else if (bil >=10 && bil<20){
        printf("Belasan");
    }
    else if (bil > 19 && bil<100){
        printf("Puluhan");
    }
    else printf("Anda Menginput Melebihi Limit Bilangan");
    return 0;
}